### PR TITLE
Updating IMU comparison and schematics

### PR DESCRIPTION
**Pull request by Aed-1**
Updating IMU comparison and schematics to include 3 LSM6DSO, LSM6DSV and LCM-45686. also updated alot of outdated info on the schematics and IMU comparison section.

### CODE REVIEW

File: src/diy/imu-comparison.md [hunk 1]
```diff
@@ -4,8 +4,11 @@ A survey was held within the DIY community for a clearer picture of the current
 This page is based on user feedback and weighed against the survey, with scores showing the average answer given to the chip.
 Following completing your own trackers, you can complete an extended version of that original survey yourself, or view previous responses on the [SlimeVR Experience Survey page](../misc/survey.md).
 
-## Index
+## Index (in order of best to worst)
```
Comment (Eirenliel) [inline src/diy/imu-comparison.md:7]:
```suggestion
## Index (in order of best to worst with current firmware)
```

File: src/diy/imu-comparison.md [hunk 2]
```diff
@@ -22,9 +25,14 @@ These factors are meant to give a quick indication as to what to expect from var
 For clarification purposes: If 3 out of 10 chips are dead on arrival or die during early use, we refer to that as poor build quality.
 
 ## General Recommendations
-At the moment, the BMI160 (stable) and BMI270 (experimental) are the best price-to-performance options, far surpassing other IMUs such as the MPU6050. Neither BMI requires a stable magnetic environment, making them a suitable option for many more people than 9DOF IMUs, such as the MPU9250 or ICM20948, or even the MPU6050+QMC5883L. BNO085s, while offering greater reset times, come at a significant price premium.
+At the moment there is no general IMU recomended for all price ranges. although below there will be a short, general guide on what to get depending on your price range:
```
Comment (Eirenliel) [inline src/diy/imu-comparison.md:28]:
```suggestion
At the moment there is no general IMU recomended for all price ranges. Although below there will be a short, general guide on what to get depending on your price range:
```

File: src/diy/imu-comparison.md [hunk 3]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
 
 |Reset time |Cost |Availability|Build quality|
 |:---------:|:---:|:----------:|:-----------:|
-|30 - 45 min|~$13 |Sufficient  |Excellent    |
+|45-60 min|~$10 |Sufficient  |Excellent    |
```
Comment (Eirenliel) [inline src/diy/imu-comparison.md:45]:
```suggestion
|30-45 min|~$11 |Sufficient  |Excellent    |
```

Idk where you got BNO085 for $10. The cheapest is $11 on our site. Also the drift time is very subjective and there are a lot of arguments, so I wouldn't recommend increasing it for BNO just yet. We really need a well-defined measurement system for drift time...

**Review by Eirenliel — APPROVED** [review]

**Comment by Aed-1** [comment]
I added all siren's suggestions as I feel they are all fair

File: src/diy/imu-comparison.md [hunk 4]
```diff
@@ -22,9 +25,14 @@ These factors are meant to give a quick indication as to what to expect from var
...
 
 ## General Recommendations
-At the moment, the BMI160 (stable) and BMI270 (experimental) are the best price-to-performance options, far surpassing other IMUs such as the MPU6050. Neither BMI requires a stable magnetic environment, making them a suitable option for many more people than 9DOF IMUs, such as the MPU9250 or ICM20948, or even the MPU6050+QMC5883L. BNO085s, while offering greater reset times, come at a significant price premium.
+At the moment there is no general IMU recomended for all price ranges. Although below there will be a short, general guide on what to get depending on your price range:
+
+    low range budget - BMI270 or LSM6DSO
+
+    mid range budget - ICM-45686 - LSM6DSV
+
+    high range budget - BNO085
```
Comment (kounocom) [inline src/diy/imu-comparison.md:34]:
```suggestion
    Low range budget - BMI270 or LSM6DSO

    Mid range budget - ICM-45686 or LSM6DSV

    High range budget - BNO085
```

File: src/diy/imu-comparison.md [hunk 5]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
-|Reliable      |Extra wiring required compared to MPUs |
+|Reliable      |Big                                    |
 |Smooth        |                                       |
 
 > Please note, if you are looking for BNO085s SlimeVR may have [spares for sale that can be found here](https://shop.slimevr.dev/products/slimevr-imu-module-bno085).
 
+---
+## LSM6DSV
+The LSM6DSV is a very good IMU for DIY SlimeVR.
+it seems to perform just as good as the BNO for a lower cost!
```
Comment (kounocom) [inline src/diy/imu-comparison.md:60]:
```suggestion
It seems to perform just as well as the BNO for a lower cost.
```

File: src/diy/imu-comparison.md [hunk 6]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
+|--------------|---------------------------------------|
+|Accurate      |Expensive                              |
+|Reliable      |Experimental                           |
+|Smooth        |                                       |
+
+
+---
+## ICM-45686
+The ICM-45686 is a potentially very good and <b>very experimental</b> IMU for DIY SlimeVR.
+So far it is shown to be testing on around the same level as the BNO/DSV but has only been tested by a few people so results are not entirely conclusive yet
```
Comment (kounocom) [inline src/diy/imu-comparison.md:79]:
```suggestion
So far it is shown to be testing on around the same level as the BNO/DSV but has only been tested by a few people so results are not entirely conclusive yet.
```

File: src/diy/imu-comparison.md [hunk 7]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
+|--------------|-------------------------------------------------------------------------|
+|Cheap         |Very Experimental                                                        |
+|Reliable      |Only available in module form from a single source                       |
+|Smooth        |                                                                         |
+
+
+---
+## LSM6DSO
+The LSM6DSO is a good IMU for DIY SlimeVR.
+it has been shown to be slightly better than the BMI270 at a slightly higher cost.
```
Comment (kounocom) [inline src/diy/imu-comparison.md:98]:
```suggestion
It has been shown to be slightly better than the BMI270 at a slightly higher cost.
```

File: src/diy/imu-comparison.md [hunk 8]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
+|:---------:|:---:|:----------:|:-----------:|
+|25-35 min|~$3.5 |Sufficient  |Great         |
+
+Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>
+
+|Pros          |Cons                                                                     |
+|--------------|-------------------------------------------------------------------------|
+|Cheap         |Experimental                                                             |
+|Reliable      |Only available in module form from a single source                       |
+|Smooth        |Poor Sensitivity                                                         |
```
Comment (kounocom) [inline src/diy/imu-comparison.md:110]:
```suggestion
|Smooth        |Poor sensitivity                                                         |
```

File: src/diy/imu-comparison.md [hunk 9]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
 Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-o" ></i>
 
 |Pros                   |Cons                                                            |
 |-----------------------|----------------------------------------------------------------|
-|Cheap                  | [Requires manual calibration the first time](#imu-calibration) |
-|Reliable               | Only available in module form from a single source             |
-|Smooth                 | Experimental, still in testing                                 |
-|Single time calibration| Performance data not taken from survey                         |
+|Cheap                  | Only available in module form from a single source             |
+|Reliable               | Experimental                                                   |
```
Comment (kounocom) [inline src/diy/imu-comparison.md:126]: I don't know if BMI270 should still be considered experimental, after an entire year of being available.

File: src/diy/imu-comparison.md [hunk 10]
```diff
@@ -34,60 +42,120 @@ They are reliable and stable chips, but at the time of writing, they remain diff
...
-It is an easily available chip with decent performance and good reliability.
 
-It does not have a magnetometer, but external chips such as QMC5883L/HMC5883L can be used,
-in the same way [as with MPU](#mpuqmc5883l). Like any other setup with magnetometers, this is highly experimental.
-Reset times and yaw accuracy with a magnetometer will depend on your build quality and magnetic environment.
+<b>Not recommended for new designs.</b>
+
+
+The BMI160 is currently not recomended for new slime trackers.
+it is very outdated with alot of newer IMUs surpassing it in price to performance
```
Comment (kounocom) [inline src/diy/imu-comparison.md:137]:
```suggestion
It is very outdated with a lot of newer IMUs surpassing it in price to performance.
```

File: src/diy/tracker-schematics.md [hunk 11]
```diff
@@ -2,15 +2,18 @@
 
 ## Wemos D1 Mini
 
-* IMU
+* IMU's ranked from best - worst
```
Comment (kounocom) [inline src/diy/tracker-schematics.md:5]:
```suggestion
* IMUs ranked from best - worst
```

File: src/diy/tracker-schematics.md [hunk 12]
```diff
@@ -2,15 +2,18 @@
...
+  - <input id="ICM45" type="radio" name="d1-imu" value="ICM45"> <label for="ICM45">ICM-45686</label> - great and cheaper than DSV. <b>VERY Experimental</b>
+  - <input id="DSO" type="radio" name="d1-imu" value="DSO"> <label for="DSO">LSM6DSO</label> - good but a little bit more expensive than BMI270. <b>Experimental</b>
+  - <input id="bmi270" type="radio" name="d1-imu" value="bmi270"> <label for="bmi270">BMI270</label> - better than BMI160 while still being cheap.
+  - <input id="bmi160" type="radio" name="d1-imu" value="bmi160"> <label for="bmi160">BMI160</label> - very cheap but not very good performance <b>DO NOT USE!</b>
+  - <input id="mpu" type="radio" name="d1-imu" value="mpu"> <label for="mpu">MPU6050</label> - Cheap but drifts quite a bit. <b>DO NOT USE!</b>
+  - <input id="mpu9250" type="radio" name="d1-imu" value="mpu9250"> <label for="mpu9250">MPU9250 (GY-91)</label> - Not terrible but plagued with fakes. <b>DO NOT USE!</b>
+  - <input id="qmc" type="radio" name="d1-imu" value="qmc"> <label for="qmc">MPU6050 + QMC5883L</label> - <b>Experimental</b> cheaper MPU9250 equivalent. <b>DO NOT USE!</b>
```
Comment (kounocom) [inline src/diy/tracker-schematics.md:16]:
```suggestion
  - <input id="DSV" type="radio" name="d1-imu" value="DSV"> <label for="DSV">LSM6DSV</label> - Very good and slightly cheaper than BNO <b>Experimental</b>
  - <input id="ICM45" type="radio" name="d1-imu" value="ICM45"> <label for="ICM45">ICM-45686</label> - Great and cheaper than DSV. <b>VERY Experimental</b>
  - <input id="DSO" type="radio" name="d1-imu" value="DSO"> <label for="DSO">LSM6DSO</label> - Good but a little bit more expensive than BMI270. <b>Experimental</b>
  - <input id="bmi270" type="radio" name="d1-imu" value="bmi270"> <label for="bmi270">BMI270</label> - Better than BMI160 while still being cheap.
  - <input id="bmi160" type="radio" name="d1-imu" value="bmi160"> <label for="bmi160">BMI160</label> - Very cheap but not very good performance <b>DO NOT USE!</b>
  - <input id="mpu" type="radio" name="d1-imu" value="mpu"> <label for="mpu">MPU6050</label> - Cheap but drifts quite a bit. <b>DO NOT USE!</b>
  - <input id="mpu9250" type="radio" name="d1-imu" value="mpu9250"> <label for="mpu9250">MPU9250 (GY-91)</label> - Not terrible but plagued with fakes. <b>DO NOT USE!</b>
  - <input id="qmc" type="radio" name="d1-imu" value="qmc"> <label for="qmc">MPU6050 + QMC5883L</label> - <b>Experimental</b> cheaper MPU9250 equivalent. <b>DO NOT USE!</b>
```

**Review by kounocom — COMMENTED** [review]
New IMUs should also get added to diy/components-guide

**Review by kounocom — APPROVED** [review]
I'd love some more discussion about whether BMI270 should remain as experimental, but otherwise LGTM. If merged, I'll submit my own PR adding these new IMUs to components-guide

**Comment by Aed-1** [comment]
> I'd love some more discussion about whether BMI270 should remain as experimental, but otherwise LGTM. If merged, I'll submit my own PR adding these new IMUs to components-guide

yea i pretty much 100% agree that the BMI270 shouldnt be experimental and i must have missed your first comment about it lol, i will remove the experimental bit shortly. i also completely forgot about the components guide and i will have a look at adding them myself to save you the effort lmao

**Review by kounocom — APPROVED** [review]

**Pull request merged**